lms/djangoapps/ccx/api/v0/serializers.py

```diff
@@ -42,3 +44,10 @@ def get_ccx_course_id(obj):
         Getter for the CCX Course ID
         """
         return unicode(CCXLocator.from_course_locator(obj.course.id, obj.id))
+
+    @staticmethod
+    def get_course_modules(obj):
+        """
+        Getter for the Course Modules. The list is stored in a compressed field.
+        """
+        return obj.structure or []
```

Comment:
actually `.structure` is the model `@property` method that takes care of parsing the JSON string stored in the `structure_json` attribute